integ-test/src/test/java/org/opensearch/sql/ppl/WhereCommandIT.java

```diff
@@ -110,4 +112,22 @@ public void testRelevanceFunction() throws IOException {
                 TEST_INDEX_BANK));
     verifyDataRows(result, rows("Hattie"));
   }
+
+  @Test
+  public void testMatchPhraseFunction() throws IOException {
+    JSONObject result =
+            executeQuery(
+                    String.format(
+                            "source=%s | where match_phrase(firstname, 'Hattie') | fields firstname", TEST_INDEX_BANK));
+    verifyDataRows(result, rows("Hattie"));
+  }
+
+  @Test
+  public void testMathPhraseWithSlop() throws IOException {
+    JSONObject result =
+            executeQuery(
+                    String.format(
+                            "source=%s | where match_phrase(firstname, 'Hattie', slop = 2) | fields firstname", TEST_INDEX_BANK));
+    verifyDataRows(result, rows("Hattie"));
+  }
```

Comment:
Test should actually use phrases for search, otherwise we can't verify that `match_phrase` works as expected.

Reply:
Fixed. See [7c3ff01](https://github.com/Bit-Quill/opensearch-project-sql/pull/48/commits/7c3ff011ef2e3fd6a7c81df508d3e6dadb6288eb)